### PR TITLE
fixes #118 - fixes Loklak query language app

### LINKED ISSUE #118: Fix the loklak query language app
The loklak query language app is not showing any result.
The page title 'Loklak Query Language' is partially getting hidden under the navbar.
Also it would be nice if there is an option to directly copy the generated URL and the generated the result.

### PR DESCRIPTION
**Pull request by djmgit**
Now the app is showing result. The header is no more hidden under the navbar.
Added option to directly copy the generated URL and the result just by clicking one button.

### CODE REVIEW

**Comment by djmgit** [comment]
demo link : 

http://loklakapps.surge.sh/LQL/

File: LQL/index.html [hunk 1]
```diff
@@ -52,7 +52,9 @@
 
       <div class="container-fluid">
       <div class="row">
-      <center><p><h3>Loklak Query Language</h3></p></center>
+      <div id="header" style="margin-top: 60px; margin-bottom: 60px;">
```
Comment (yukiisbored) [inline LQL/index.html]: I think it's better to put this fix into it's own commit instead of combining it with the loklak url fix.

**Comment by yukiisbored** [comment]
It's hard for me to see what lines does what if you combine them into one huge commit like this. Please break it into multiple commits for each issues regarding LQL.

**Comment by djmgit** [comment]
@yukiisbored Thanks for reviewing, I shall create a separate issue for the changes other than the URL change which I have done.

**Comment by mariobehling** [comment]
Any chance to get this PR building with Travis?

**Comment by yukiisbored** [comment]
@mariobehling Yeah, just fix the issues reported by coala

**Comment by djmgit** [comment]
I have split this issue into different issues, PRs have been merged, so closing this issue.
